source/features/pr-branch-auto-delete.tsx

```diff
@@ -57,12 +54,11 @@ void features.add(import.meta.url, {
 	asLongAs: [
 		pageDetect.isPRConversation,
 		pageDetect.isOpenPR,
-		canCreateRelease,
+		userHasPushAccess,
```

Comment:
See note.

Technically this should be:
```suggestion
		async () => {
			await elementReady('PR header')
			const pr = getBranches();
			return userHasPushAccess(pr.head.nameWithOwner)
		},
```

Not sure it's worth it though. I'll just leave a note for now